ZAM_fnc_EZMLiteV2.1.1.sqf

```diff
@@ -6171,20 +6186,28 @@ MAZ_EZM_fnc_initFunction = {
 						"",
 						1
 					]
+				],
+				[
+					"TOOLBOX",
+					"Zeus Sees Cutscene",
+					[
+						false,
+						["No", "Yes"]
```

Comment:
If you're only wanting to use "YES" and "NO" as the strings you can forgo the ["No", "Yes"] and just make the content type have a subtype: `"TOOLBOX:YESNO"`. Not an important change, nor does it really matter.